languages/csharp/concepts/enums/about.md

```diff
@@ -19,11 +19,26 @@ Users.WithStatus(1)
 Users.WithStatus(Status.Active)
 ```
 
-For someone reading the code, the second (enum) version will be easier to comprehend.
+For someone reading the code, the second version (with enum) will be easier to comprehend.
 
-You should always consider using an enum whenever you want to model something as a boolean. Besides the aforementioned readability benefits, enums have another advantage over booleans: new values can always be added to an enum, whereas a boolean value will only ever be `true` or `false`. Using an enum is thus more future proof.
+## Numeric type Enums
 
-Note that while one _can_ cast integer values to an enum, doing so can lead to unexpected results when the integer value doesn't map to any enum value:
+The default numeric type of the values is `int`, it can be changed by explicitly specifying any other [integral numeric][integral numeric] type.
```

Comment:
```suggestion
While the default numeric type of the values is `int`, it can be changed by explicitly specifying any other [integral numeric][integral numeric] type.
```